test/EIP20CoGateway/_confirmStakeIntentRawTx.js

```diff
@@ -110,20 +98,14 @@ describe('EIP20CoGateway._confirmStakeIntentRawTx()', () => {
         stakeParams.hashLock,
         stakeParams.blockHeight,
         stakeParams.storageProof,
-      )
-      .catch((exception) => {
-        assert.strictEqual(
-          exception.message,
-          expectedErrorMessage,
-          `Exception reason must be "${expectedErrorMessage}"`,
-        );
-      });
+      ),
+      `Invalid beneficiary address: ${undefined}.`,
```

Comment:
Is it different from this?
```suggestion
      'Invalid beneficiary address: undefined.',
```

Reply:
No, it is not different, it is just the preference.
I have changed it for all the tests.